www/templates/account/billing/update-payment-confirm-address.php

```diff
@@ -78,4 +78,25 @@
         <input type="hidden" name="type" value="update-payment-method-confirm-billing" />
         <button class="yellow pill-button" type="submit">Confirm Billing Address</button>
     </form>
+    <script defer src="/assets/js/country-list/country-list.js?v=<?= constant('VER_JS_COUNTRY_LIST') ?>"></script>
```

Comment:
@mjkozicki this is same script we have in signup page

Reply:
@alectejada please copy the exact scirpt, why you need the php  tag?

Reply:
this is the script that is on the billing-address-form.php is the one we use on signup.